.github/workflows/continuous_integration.yml

```
@@ -7,301 +7,296 @@
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        compiler: [{cc: clang, cxx: clang++}, {cc: gcc, cxx: g++}]
+        cmake_build_type: [Debug, Release]
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - uses: lukka/get-cmake@8be6cca406b575906541e8e3b885d46f416bba39 # v3.27.7
+      - uses: actions/setup-python@65d7f2d534ac1bc67fcd62888c5f4f3d2cb2b236 # v4.7.1
+        with:
+          python-version: '3.7'
+      - name: Setup ccache
+        uses: hendrikmuhs/ccache-action@6d1841ec156c39a52b1b23a810da917ab98da1f4 # v1.2.10
+        with:
+          key: ubuntu-22-${{ matrix.cmake_build_type }}-${{ matrix.compiler.cc }}-${{matrix.compiler.cxx}}
+      - name: Install GoogleTest
+        run: |
+          # check out pre-breakage version of googletest; can be deleted when
+          # issue 3128 is fixed
+          # git clone --depth=1 https://github.com/google/googletest.git External/googletest
+          mkdir -p External/googletest
+          cd External/googletest
+          git init
+          git remote add origin https://github.com/google/googletest.git
+          git fetch --depth 1 origin 0c400f67fcf305869c5fb113dd296eca266c9725
+          git reset --hard FETCH_HEAD
+          cd ../..
+      - name: Update Glslang Sources
+        run: ./update_glslang_sources.py
+      - name: Configure
+        run: cmake -S . -B build -D CMAKE_BUILD_TYPE=${{ matrix.cmake_build_type }}
+        env:
+          CC: ${{matrix.compiler.cc}}
+          CXX: ${{matrix.compiler.cxx}}
+          CMAKE_GENERATOR: Ninja
+          CMAKE_C_COMPILER_LAUNCHER: ccache
+          CMAKE_CXX_COMPILER_LAUNCHER: ccache
+      - name: Build
+        run: cmake --build build
+      - name: Install
+        run: cmake --install build --prefix build/install
+      - name: Test
+        run: ctest --output-on-failure --test-dir build
+      - name: Test (standalone)
+        run: cd Test && ./runtests
```

If I understand the original problem this patch is addressing, then is it possible that the Test directory does not exist and the runtests script can silently fail?

---

Oops... this is linux... sorry. Disregard.